.changes/fix-titlebar-transparent.md

```diff
@@ -0,0 +1,5 @@
+---
+"tao": patch
+---
+
+Fix macOS `WindowBuilderExtMacOS::with_titlebar_transparent`
```

Comment:
I don't think it was broken in the first place, because it seems like users needed to just pair it with `WindowBuilderExtMacOS::with_fullsize_content_view` in wry/tauri apps.